files/image_config/monit/memory_checker

```diff
@@ -94,6 +96,31 @@ def check_memory_usage(container_name, threshold_value):
         sys.exit(4)
 
 
+def get_running_container_names():
+    """Retrieves names of running containers by talking to the docker daemon.
+
+    Args:
+        None.
+
+    Returns:
+        running_container_names: A list indicates names of running containers.
+    """
+    running_container_names = []
+    docker_client = docker.DockerClient(base_url='unix://var/run/docker.sock')
```

Comment:
@yozhao101 do we need handle the case when docker engine is not started? Let's say memory checker starts simultaneously with docker service

Reply:
`memory_checker` script will be ran by Monit periodically and Monit is delayed to run this script after device is up 300 seconds. 

Docker daemon will be started by systemd immediately once device is booted/rebooted.

If docker daemon crashed or is not started successfully, then `memory_checker` will exit and log an error message into syslog.

Reply:
@yozhao101 ok